diff -aurN slim-1.3.5.orig/panel.cpp slim-1.3.5/panel.cpp
--- slim-1.3.5.orig/panel.cpp	2013-02-10 14:24:57.182705119 +0100
+++ slim-1.3.5/panel.cpp	2013-02-10 14:29:41.715070379 +0100
@@ -23,7 +23,7 @@
 	Root = root;
 	cfg = config;
 
-	session = "";
+	session = cfg->nextSession(session);
 
 	/* Init GC */
 	XGCValues gcv;
@@ -184,7 +184,7 @@
 }
 
 void Panel::ClearPanel() {
-	session = "";
+	session = cfg->nextSession(session);
 	Reset();
 	XClearWindow(Dpy, Root);
 	XClearWindow(Dpy, Win);
